### DISCUSSION TITLE
Platform M+ and D display setup

### DISCUSSION THREAD

**Discussion by calvin006** [helgoboss/helgobox#446] [Q&A] [answered]
I'm just getting started with Realearn, using with a Platform M+. With the "DAW Control" preset, I seem to have the basic functions working. However, I am not getting any information on the LCD display (the Platform D attachment). It just says "MCP Mode Selected" where it should be displaying the track names and pan/fader values. Any tips for getting the display working? Apologies if I'm missing something obvious. Thanks.

**Comment by helgoboss** [comment]
Hi. Support for LCDs hasn't made it very far yet in ReaLearn because I never felt it's urgent. However, lately I have been thinking about it and plan to implement it, soon. There's an issue for it now: #448. This issue also describes what's already possible in terms of LCDs (but as I said, it's not much).

**Reply by calvin006** [reply]
Thanks. I'll keep an eye out for implementation.

I"m mostly interested in basic mixing functionality and the display is useful to quickly confirm what track I'm controlling.

Maybe you know... is there a way to display the current selected fader bank in Reaper? For instance, in Logic there's a colored bar In the Logic mix window that's 8 faders long that shows you what bank of faders are being controlled. Reaper doesn't seem to have this. Without that and without scribble strip info on my display, it's hard to know what tracks I'm controlling.

**Reply by helgoboss** [reply]
ReaLearn v2.11.0-pre.3 supports LCDs. Would be great if you could give it a try. I have a Platform M+ but unfortunately without display component. Here's the announcement: https://forum.cockos.com/showpost.php?p=2487210&postcount=1618

**Comment by calvin006** [comment]
Hi there, just checking back about Platform D display functionality. I remember you mentioned building this into a future preset update. I just updated to the latest Realearn and Platform M+ preset but not seeing anything on the display yet. Just wanted to make sure I'm not doing something wrong. Thanks.

**Reply by helgoboss** [reply, marked as answer]
You need the latest Platform M+ controller preset AND combine it with the latest DAW control main preset.

If this doesn't work for some reason, try the Mackie Control controller preset.

**Reply by calvin006** [reply]
Everything was updated. I had to change the presets to <None> and then back again. That seemed to switch it over to the updates. Now I've got text display. Working great so far, thanks!

**Reply by helgoboss** [reply]
Oh yes. It doesn't refresh them automatically, otherwise that could lead to really unexpected changes of existing projects just by synchronizing ReaPack.
